static/js/components/dashboard/courses/GradeDetailPopup.js

```diff
@@ -3,57 +3,87 @@ import Dialog from 'material-ui/Dialog';
 import React from 'react';
 import R from 'ramda';
 import Icon from 'react-mdl/lib/Icon';
+import moment from 'moment';
 
 import type {
   Course,
   CourseRun,
+  ProctoredExamResult,
 } from '../../../flow/programTypes';
 import { formatGrade } from '../util';
 import {
   STATUS_NOT_PASSED,
   STATUS_PASSED,
+  DASHBOARD_FORMAT,
 } from '../../../constants';
+import type { GradeType } from '../../../containers/DashboardPage';
+import { EDX_GRADE } from '../../../containers/DashboardPage';
 
-const runGrade = (courseRun: CourseRun, isBestGrade: boolean): string|React$Element<*> => {
-  if (courseRun.final_grade) {
+const formatExamGrade = percentageGrade => percentageGrade * 100;
```

Comment:
`formatGrade` expects a grade which is already on the 0-100 scale, whereas the `percentage_grade` number we get for the exam grades is from 0-1, so we need to mutiply it by 100 before we pass it to the `formatGrade` function. If you look below, where this is called, the result of the `formatExamGrade` function is passed to `rowGrade`, which calls `formatGrade` before displaying the grade.